File: simplevlcplayer/src/main/java/com/masterwok/simplevlcplayer/dagger/ActivityModule.java
package com.masterwok.simplevlcplayer.dagger;

import com.masterwok.simplevlcplayer.activities.MediaPlayerActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
abstract class ActivityModule
{
    @ContributesAndroidInjector
    abstract MediaPlayerActivity contributeMainActivity();
}

File: simplevlcplayer/src/main/java/com/masterwok/simplevlcplayer/dagger/modules/ActivityModule.kt
package com.masterwok.simplevlcplayer.dagger.modules

import com.masterwok.simplevlcplayer.activities.MediaPlayerActivity
import dagger.Module
import dagger.android.ContributesAndroidInjector


@Module
@Suppress("unused")
abstract class ActivityModule {

    @ContributesAndroidInjector()
    abstract fun contributeMediaPlayerActivity(): MediaPlayerActivity

}
